src/parser/toplevel.lisp

```diff
@@ -2069,8 +2071,34 @@ consume all attributes")))
                  :message "Malformed struct field"
                  :primary-note "expected field type")))
 
```

Comment:
Write something like so that the rest of the parser doesn't need to worry about the docstring:

```commonlisp
(multiple-value-bind (docstring rest-field)
     (if (and (cst:atom (cst:first (cst:rest form)))
              (stringp (cst:raw (cst:first (cst:rest form)))))
           (values (cst:raw (cst:first (cst:rest form))) (cst:rest (cst:rest form))
           (values nil (cst:rest form)))
```